src/citrine/informatics/design_candidate.py

```diff
@@ -117,8 +117,10 @@ class DesignCandidate(Serializable["DesignCandidate"]):
     This class represents the candidate computed by a design execution.
     """
 
+    candidate_id = properties.UUID('id')
+    """:UUID: unique external Citrine id of the material"""
```

Comment:
Ya know, until just this moment, I thought a candidate was made up of materials. So thanks for correcting me that they're synonyms!